File: javascript/resistor-color-duo/resistor-color-duo.js
export const COLORS = [
        'black', 'brown', 'red', 'orange', 'yellow',
        'green', 'blue', 'violet', 'grey', 'white'];

export const decodedValue = ([firstColor, secondColor, ...rest]) => 
                        COLORS.indexOf(firstColor) * 10 + COLORS.indexOf(secondColor);

File: python/resistor-color-duo/resistor_color_duo.py

lookup ={
    "black": 0,
    "brown": 1,
    "red": 2,
    "orange": 3,
    "yellow": 4,
    "green": 5,
    "blue": 6,
    "violet": 7,
    "grey": 8,
    "white": 9
}

def value(colors):
    first_color, second_color, *_rest = colors

    return lookup[first_color] * 10 + lookup[second_color]
